src/shared/assets/icons.js

```diff
@@ -14,4 +16,21 @@ async function geticons() {
   return icons;
 }
 
-module.exports = { geticons };
+async function selecticon() {
```

Comment:
Typo in selection function.

Reply:
Ah, It's selecticon not selection.

Reply:
Everything looks good. Am I missing something?

Reply:
Why not make the function name `selectIcon` that would be easier to read or else anyone would think its a typo.

Reply:
Yes, I was about to do that but then I saw `geticons` you named named as that

Reply:
First of all remove geticons from export since we are using it only in the icons.js file. And since we are calling it only once we  can change its name maybe. WDYT?

Reply:
I already deleted the`geticons ` export everywhere, it's still showing?

Reply:
I will chnage all the naming conventions right now to camel case. I didn't point it out earlier because you did in that way :)

Reply:
In icons.js file we are doing module export for geticons and selecticon. We don't need geticons anywhere else so we don't have to export it.

Reply:
> I will change all the naming conventions right now to camel case. I didn't point it out earlier because you did in that way :)

You know what's interesting. I did it because you originally named all your functions like that so I thought I should keep it that way. 😆

Reply:
> > I will change all the naming conventions right now to camel case. I didn't point it out earlier because you did in that way :)
> 
> You know what's interesting. I did it because you originally named all your functions like that so I thought I should keep it that way. 😆

DANG. You should have pointed it out 😆

Reply:
> In icons.js file we are doing module export for geticons and selecticon. We don't need geticons anywhere else so we don't have to export it.

My bad my bad. I misunderstood it.

Reply:
Yeah that was my fault, I should have told u but because of all other complex things we were working on I forgot about it probably. It's ok you get to make an extra contribution today because of it so nothing wrong with that I guess 😉

Reply:
> You know what's interesting. I did it because you originally named all your functions like that so I thought I should keep it that way. 😆

Wait Wait. I didn't. I have done correctly. See below

https://github.com/Pradumnasaraf/LinkFree-CLI/tree/1977013174d5a288090aee1b018a04683105ad1

Reply:
See this: https://github.com/Pradumnasaraf/LinkFree-CLI/blob/1977013174d5a288090aee1b018a04683105ad1c/src/index.js

The `addlinkstojson` function at the end.

Reply:
Leave it. My fault 🤣

Reply:
Good otherwise I would have gone into my competitive mode to make you agree 🤣

Reply:
Haha, forgive me. 

You can review it now.